llpc/test/shaderdb/hlsl/PipelineHlsl_TestRuntimeArrayPacking.pipe

```diff
@@ -0,0 +1,430 @@
+userDataNode[0].offsetInDwords = 0
+userDataNode[0].sizeInDwords = 1
+userDataNode[0].indirectUserDataCount = 0
+userDataNode[1].visibility = 16
+userDataNode[1].type = StreamOutTableVaPtr
+userDataNode[1].offsetInDwords = 2
+userDataNode[1].sizeInDwords = 1
+userDataNode[2].visibility = 12
+userDataNode[2].type = DescriptorTableVaPtr
+userDataNode[2].offsetInDwords = 4
+userDataNode[2].sizeInDwords = 1
+userDataNode[2].next[0].type = DescriptorBuffer
+userDataNode[2].next[0].offsetInDwords = 0
+userDataNode[2].next[0].sizeInDwords = 4
+userDataNode[2].next[0].set = 0x00000000
+userDataNode[2].next[0].binding = 0
+userDataNode[2].next[1].type = DescriptorBuffer
+userDataNode[2].next[1].offsetInDwords = 4
+userDataNode[2].next[1].sizeInDwords = 4
+userDataNode[2].next[1].set = 0x00000000
+userDataNode[2].next[1].binding = 1
+
+[GraphicsPipelineState]
+topology = VK_PRIMITIVE_TOPOLOGY_PATCH_LIST
+provokingVertexMode = VK_PROVOKING_VERTEX_MODE_FIRST_VERTEX_EXT
+patchControlPoints = 1
+deviceIndex = 0
+disableVertexReuse = 0
+switchWinding = 0
+enableMultiView = 0
+depthClipEnable = 1
+rasterizerDiscardEnable = 1
+perSampleShading = 0
+numSamples = 1
+pixelShaderSamples = 0
+samplePatternIdx = 0
+usrClipPlaneMask = 0
+alphaToCoverageEnable = 0
+dualSourceBlendEnable = 0
+nggState.enableNgg = 1
+nggState.enableGsUse = 0
+nggState.forceCullingMode = 0
+nggState.compactMode = NggCompactDisable
+nggState.enableVertexReuse = 0
+nggState.enableBackfaceCulling = 1
+nggState.enableFrustumCulling = 0
+nggState.enableBoxFilterCulling = 0
+nggState.enableSphereCulling = 0
+nggState.enableSmallPrimFilter = 1
+nggState.enableCullDistanceCulling = 0
+nggState.backfaceExponent = 0
+nggState.subgroupSizing = Auto
+nggState.primsPerSubgroup = 256
+nggState.vertsPerSubgroup = 256
+dynamicVertexStride = 0
+enableUberFetchShader = 0
+enableEarlyCompile = 0
+options.includeDisassembly = 0
+options.scalarBlockLayout = 1
+options.includeIr = 0
+options.robustBufferAccess = 0
+options.reconfigWorkgroupLayout = 0
+options.shadowDescriptorTableUsage = Disable
+options.shadowDescriptorTablePtrHigh = 2
+options.extendedRobustness.robustBufferAccess = 0
+options.extendedRobustness.robustImageAccess = 0
+options.extendedRobustness.nullDescriptor = 0
+options.optimizeTessFactor = 0
+
+
+;Compiler Options: amdvlk -enable-errs=1 -enable-outs=1 -log-file-outs=D:\/Jira\swdev333864/spvLogInfoLlpc.txt -log-file-dbgs=debugLogInfo.txt -enable-pipeline-dump -pipeline-dump-dir=D:\/Jira\swdev333864 -executable-name=deqp-vk.exe -shader-cache-file-dir=C:\Users\xuechen2\AppData\Local -shader-cache-mode=0 -subgroup-size=64
+;PipelineOptimizer
+;Vertex   Shader Profile Key: 0x1C9ABA8F9404FCF7FA84FA5F11B64FD4,
+;Tessellation control Shader Profile Key: 0x586E53D4AA4C92DF281677352C8270DD,
+;Tessellation evaluation Shader Profile Key: 0x82B4BACBF302B7567083656B51E09C6A,
+
+[CompileLog]
+(null) (size = 0 bytes)
+
+.AMDGPU.disasm (size = 5707 bytes)
+_amdgpu_gs_main:
```

Comment:
Don't attach compile log (ISA, register config, binary code) in the .pipe file. It will make the file become longer unnecessarily.

Reply:
Thanks for your reminder. Fixed.